index.js

```diff
@@ -1,11 +1,15 @@
 const express = require('express');
 
+const fs = require('fs');
+let rawdata = fs.readFileSync('package.json');
+const packageFile = JSON.parse(rawdata);
+
 const app = express();
 const PORT = 8080;
 
-app.get('/', (req, res) => {
-  res.send('Test EndPoint');
-});
+app.get('/info', (req, res) => {
+    res.json(packageFile);
```

Comment:
We don't need to send back all the information in the package file. Just the required details in the task are enough